index.html

```diff
@@ -52,18 +54,22 @@
       background: var(--background-plus);
     }
 
-    [dir=rtl] gaia-list.contacts .face {
+    gaia-list.contacts .gaia-item-circular > img {
+      width: 100%;
+    }
+
+    [dir=rtl] gaia-list.contacts .gaia-item-circular {
```

Comment:
moving into gaia-button with `:host-context([dir=rtl]) ::content .gaia-item-circular {` not work :(